build/Build.csproj

```diff
@@ -1,17 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-	<TargetFramework>net6.0</TargetFramework>
-	<SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
-  </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="Colors.Net" Version="1.1.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="NuGet.Versioning" Version="5.10.0" />
-    <PackageReference Include="System.IO.Abstractions" Version="2.1.0.227" />
-    <PackageReference Include="WindowsAzure.Storage" Version="9.3.2" />
-	<PackageReference Include="Microsoft.Extensions.DependencyModel" Version="7.0.0" />
-  </ItemGroup>
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
+	</PropertyGroup>
+	<ItemGroup>
+		<PackageReference Include="Colors.Net" Version="1.1.0" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+		<PackageReference Include="NuGet.Versioning" Version="6.12.1" />
+		<PackageReference Include="NuGet.Protocol" Version="6.12.1" />
+		<PackageReference Include="System.IO.Abstractions" Version="2.1.0.227" />
+		<PackageReference Include="System.Net.Http" Version="4.3.4" />
+		<PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
+		<PackageReference Include="WindowsAzure.Storage" Version="9.3.3" />

```

Comment:
@fabiocav  - Can we get remove this package - WindowsAzure.Storage? It is deprecated and I also couldn't find a reference/use of it.

Reply:
I can also upgrade System.IO.Abstractions and packages for test csproj at the sametime

Reply:
taking care of this in a separate PR.